backend/x11/output.c

```diff
@@ -600,6 +601,8 @@ struct wlr_output *wlr_x11_output_create(struct wlr_backend *backend) {
 	xcb_map_window(x11->xcb, output->win);
 	xcb_flush(x11->xcb);
 
+	output->mapped = true;
```

Comment:
I wonder if we should wait for `MapNotify` here?

Reply:
We won't get it. We only insert the window into the list after this happens.